src/components/Tool/FixPhoneNumbers/FixPhoneNumbers.tsx

```diff
@@ -83,364 +87,354 @@ const useStyles = makeStyles()(() => ({
 
 export interface ModalState {
   open: boolean;
-  personId: string;
+  personIndex: number;
   numberIndex: number;
   phoneNumber: string;
 }
 
 const defaultDeleteModalState = {
   open: false,
-  personId: '',
+  personIndex: 0,
   numberIndex: 0,
   phoneNumber: '',
 };
 
-export interface PhoneNumberData {
-  id?: string;
-  primary: boolean;
-  updatedAt: string;
-  source: string;
-  number: string;
-  destroy?: boolean;
-}
-
 export interface PersonPhoneNumbers {
-  phoneNumbers: PhoneNumberData[];
-  toDelete: PersonPhoneNumberInput[];
+  phoneNumbers: PersonPhoneNumberFragment[];
 }
 
 interface Props {
   accountListId: string;
   setContactFocus: SetContactFocus;
 }
 
+export interface FormValuesPerson extends PersonInvalidNumberFragment {
+  newPhoneNumber: PersonPhoneNumberFragment | string;
```

Comment:
I think this type is OK to just be a string, right?

```suggestion
  newPhoneNumber: string;
```